mopidy_listenbrainz/__init__.py

```diff
@@ -1,14 +1,21 @@
 import pathlib
 
 import pkg_resources
+import email.parser
 from mopidy import config, ext
 
-__version__ = pkg_resources.get_distribution("Mopidy-Listenbrainz").version
+__dist_name__ = "Mopidy-Listenbrainz"
+_dist = pkg_resources.get_distribution(__dist_name__)
```

Comment:
This doesn't support execution from within the source directory - but it also didn't before, so...